README.md

```diff
@@ -50,238 +50,99 @@ Course authors can fully define and customize the lab environment. It is only
-            "delete_age": 14,
-            "delete_attempts": 3,
-            "delete_task_timeout": 900,
-            "ssh_connect_timeout": 10,
-            "sleep_timeout": 10,
-            "js_timeouts": {
-                "status": 15000,
-                "keepalive": 30000,
-                "idle": 3600000,
-                "check": 5000
-            },
-            "providers": {
-                "default": {
-                    "type": "openstack",
-                    "os_auth_url": "",
-                    "os_auth_token": "",
-                    "os_username": "",
-                    "os_password": "",
-                    "os_user_id": "",
-                    "os_user_domain_id": "",
-                    "os_user_domain_name": "",
-                    "os_project_id": "",
-                    "os_project_name": "",
-                    "os_project_domain_id": "",
-                    "os_project_domain_name": "",
-                    "os_region_name": ""
-                },
-                "provider2": {
-                    "type": "gcloud",
-                    "gc_type": "service_account",
-                    "gc_project_id": "",
-                    "gc_private_key_id": "",
-                    "gc_private_key": "",
-                    "gc_client_email": "",
-                    "gc_client_id": "",
-                    "gc_auth_uri": "",
-                    "gc_token_uri": "",
-                    "gc_auth_provider_x509_cert_url": "",
-                    "gc_client_x509_cert_url": "",
-                    "gc_region_id": ""
-                },
-            }
-        }
-    }
-    ```
-
-7. Now install the Guacamole web app and stack supervisor scripts by cloning
-   the `hastexo_xblock` fork of edx/configuration and assigning that role to
-   the machine:
-
-    ```
-    $ git clone -b hastexo/juniper/hastexo_xblock https://github.com/hastexo/edx-configuration.git
-    $ cd edx-configuration/playbooks
-    $ ansible-playbook -c local -i "localhost," run_role.yml -e role=hastexo_xblock
-    ```
-
-8. At this point restart the LMS, the CMS, and the Celery workers, and
-   also make sure the stack management jobs are running:
-
-    ```
-    sudo /edx/bin/supervisorctl restart lms:
-    sudo /edx/bin/supervisorctl restart cms:
-    sudo /edx/bin/supervisorctl restart edxapp_worker:
-    sudo /edx/bin/supervisorctl start suspender:
-    sudo /edx/bin/supervisorctl start reaper:
-    ```
-
-9. Finally, in your course, go to the advanced settings and add the hastexo
-   module to the "Advanced Module List" like so:
-
+3. Before deploying services with Tutor, build the Docker image for
+   the `hastexo` service, and also build a custom `openedx` image:
+   ```
+   tutor images build openedx hastexo
+   ```
+   For more details about each setting, please refer to the *XBlock
+   settings* section below.
+   For more information about the plugin configurations, please refer to the
+   plugin README.
```

Comment:
This section should probably be at the end of step number 2 above.

Reply:
The first sentence, yes. The second one, I think, belongs where it is.